<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Zero_Security</class>
 <widget class="QWidget" name="Zero_Security">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1132</width>
    <height>842</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Zero Security</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLineEdit" name="path"/>
       </item>
       <item>
        <widget class="QPushButton" name="browse">
         <property name="text">
          <string>Browse</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QComboBox" name="dropbox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>25</height>
          </size>
         </property>
         <item>
          <property name="text">
           <string>Select</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>HKEY_CLASSES_ROOT</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>HKEY_CURRENT_USER</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>HKEY_LOCAL_MACHINE</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>HKEY_USERS</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>HKEY_CURRENT_CONFIG</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="cancel">
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="submit">
         <property name="text">
          <string>Submit</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QSplitter" name="splitter">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <widget class="QWidget" name="">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QTableWidget" name="path_table">
           <property name="mouseTracking">
            <bool>true</bool>
           </property>
           <property name="autoFillBackground">
            <bool>true</bool>
           </property>
           <property name="editTriggers">
            <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
           </property>
           <property name="tabKeyNavigation">
            <bool>true</bool>
           </property>
           <property name="showDropIndicator" stdset="0">
            <bool>false</bool>
           </property>
           <property name="dragDropOverwriteMode">
            <bool>false</bool>
           </property>
           <property name="alternatingRowColors">
            <bool>true</bool>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::SelectionMode::NoSelection</enum>
           </property>
           <property name="selectionBehavior">
            <enum>QAbstractItemView::SelectionBehavior::SelectRows</enum>
           </property>
           <attribute name="horizontalHeaderStretchLastSection">
            <bool>true</bool>
           </attribute>
           <column>
            <property name="text">
             <string>Path</string>
            </property>
           </column>
          </widget>
         </item>
         <item>
          <widget class="QTabWidget" name="dashboard_plot">
           <property name="currentIndex">
            <number>0</number>
           </property>
           <widget class="QWidget" name="ax1">
            <attribute name="title">
             <string>Detection Rate</string>
            </attribute>
            <layout class="QGridLayout" name="gridLayout_5"/>
           </widget>
           <widget class="QWidget" name="ax2">
            <attribute name="title">
             <string>Types of paths</string>
            </attribute>
            <layout class="QGridLayout" name="gridLayout_3"/>
           </widget>
           <widget class="QWidget" name="ax3">
            <attribute name="title">
             <string>System Categories</string>
            </attribute>
            <layout class="QGridLayout" name="gridLayout_2"/>
           </widget>
           <widget class="QWidget" name="ax4">
            <attribute name="title">
             <string>Software Categories</string>
            </attribute>
            <layout class="QGridLayout" name="gridLayout"/>
           </widget>
          </widget>
         </item>
         <item>
          <widget class="QTableWidget" name="result_table">
           <property name="mouseTracking">
            <bool>true</bool>
           </property>
           <property name="autoFillBackground">
            <bool>true</bool>
           </property>
           <property name="editTriggers">
            <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
           </property>
           <property name="tabKeyNavigation">
            <bool>true</bool>
           </property>
           <property name="showDropIndicator" stdset="0">
            <bool>false</bool>
           </property>
           <property name="dragDropOverwriteMode">
            <bool>false</bool>
           </property>
           <property name="alternatingRowColors">
            <bool>true</bool>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::SelectionMode::NoSelection</enum>
           </property>
           <property name="selectionBehavior">
            <enum>QAbstractItemView::SelectionBehavior::SelectRows</enum>
           </property>
           <property name="gridStyle">
            <enum>Qt::PenStyle::DashLine</enum>
           </property>
           <property name="sortingEnabled">
            <bool>true</bool>
           </property>
           <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
            <bool>false</bool>
           </attribute>
           <attribute name="horizontalHeaderStretchLastSection">
            <bool>true</bool>
           </attribute>
           <attribute name="verticalHeaderShowSortIndicator" stdset="0">
            <bool>false</bool>
           </attribute>
           <column>
            <property name="text">
             <string>Found</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Path</string>
            </property>
           </column>
          </widget>
         </item>
         <item>
          <widget class="QTableWidget" name="hardening_table">
           <property name="editTriggers">
            <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
           </property>
           <property name="tabKeyNavigation">
            <bool>true</bool>
           </property>
           <property name="showDropIndicator" stdset="0">
            <bool>false</bool>
           </property>
           <property name="dragDropOverwriteMode">
            <bool>false</bool>
           </property>
           <property name="alternatingRowColors">
            <bool>true</bool>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::SelectionMode::NoSelection</enum>
           </property>
           <property name="selectionBehavior">
            <enum>QAbstractItemView::SelectionBehavior::SelectRows</enum>
           </property>
           <property name="gridStyle">
            <enum>Qt::PenStyle::DashLine</enum>
           </property>
           <property name="sortingEnabled">
            <bool>true</bool>
           </property>
           <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
            <bool>false</bool>
           </attribute>
           <attribute name="horizontalHeaderStretchLastSection">
            <bool>true</bool>
           </attribute>
           <column>
            <property name="text">
             <string>Hardening</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Path</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QTableWidget" name="system_info">
        <property name="editTriggers">
         <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
        </property>
        <property name="showDropIndicator" stdset="0">
         <bool>false</bool>
        </property>
        <property name="dragDropOverwriteMode">
         <bool>false</bool>
        </property>
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::SelectionMode::NoSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectionBehavior::SelectRows</enum>
        </property>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <column>
         <property name="text">
          <string>Item</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Values</string>
         </property>
        </column>
       </widget>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="submit_hardening">
         <property name="text">
          <string>Hardening</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="hardening">
         <property name="text">
          <string>Hardening</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="check_security">
         <property name="text">
          <string>Submit</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="check_security_result">
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Dashboard</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>path</tabstop>
  <tabstop>browse</tabstop>
  <tabstop>dropbox</tabstop>
  <tabstop>submit</tabstop>
  <tabstop>path_table</tabstop>
  <tabstop>check_security</tabstop>
  <tabstop>result_table</tabstop>
  <tabstop>cancel</tabstop>
  <tabstop>check_security_result</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>cancel</sender>
   <signal>clicked()</signal>
   <receiver>path</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>928</x>
     <y>96</y>
    </hint>
    <hint type="destinationlabel">
     <x>232</x>
     <y>33</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancel</sender>
   <signal>clicked()</signal>
   <receiver>path_table</receiver>
   <slot>clearContents()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>882</x>
     <y>84</y>
    </hint>
    <hint type="destinationlabel">
     <x>620</x>
     <y>107</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancel</sender>
   <signal>clicked()</signal>
   <receiver>result_table</receiver>
   <slot>clearContents()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>916</x>
     <y>87</y>
    </hint>
    <hint type="destinationlabel">
     <x>631</x>
     <y>482</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
